doc/js_tutorials/js_setup/js_setup/js_setup.markdown

```diff
@@ -83,6 +83,9 @@ Building OpenCV.js from Source
     It requires `python` and `cmake` installed in your development environment.
 
 -#  The build script builds asm.js version by default. To build WebAssembly version, append `--build_wasm` switch.
+    By default everything is bundled into one JavaScript file by `base64` encoding the WebAssembly code. For production
+    builds you can add `--disable_single_file` which will reduce total size by writing the WebAssembly code
+    to a dedicated `.wasm` file which the generated JavaScript file will automatically load.
```

Comment:
Can this approach fix this issue as well? https://github.com/opencv/opencv/issues/21431. Which size reduction do you observe?

Reply:
* Single file: 8.74 MB `.js`
* Separate `.wasm` file: 170 kB `.js` + 6.43 MB `.wasm`

I.e. bundling the generated webassembly code with `base64` is ~32-33 % larger in total compared with separate `.wasm` file (consistent with typical `base64` encoding size increases).